test/jdk/sun/net/www/http/KeepAliveCache/B5045306.java

```diff
@@ -63,150 +66,166 @@
 
 public class B5045306 {
     static HttpServer server;
-
-    public static void main(String[] args) {
-        startHttpServer();
-        clientHttpCalls();
-    }
+    static ExecutorService executor = Executors.newSingleThreadExecutor();
 
     public static void startHttpServer() {
         try {
             server = HttpServer.create(new InetSocketAddress(InetAddress.getLocalHost(), 0), 10, "/", new SimpleHttpTransactionHandler());
```

Comment:
While we are changing this test, can you change this to `InetAddress.getLoopbackAddress()` and then when constructing the requet URI, use `URIBuilder.loopback()`?

Reply:
Done